configs/common/packages.yaml

```diff
@@ -9,290 +9,255 @@
         glu: [openglu]
         jpeg: [libjpeg-turbo]
         lapack: [openblas]
+        pkgconfig: [pkg-config]
         yacc: [bison]
         zlib-api: [zlib-ng]
     #
     awscli-v2:
-      require: "~examples"
+      require: '~examples'
     bacio:
-      version: ['2.4.1']
+      require: '@2.4.1'
     bison:
-      version: ['3.8.2']
-    # 1.85 incompatible with ecflow@5.11.4 - use latest "good version"
+      require: '@3.8.2'
```

Comment:
This is not going to work. When we move to oneapi, we need to use an external bison since oneapi doesn't build bison correctly (conflict in the package). We'll have to accept whatever `bison@3` that is found on the system.

Reply:
or maybe `::` for the external package? I'd prefer just setting `require: '@3'` if possible

Reply:
For now, I'm setting it to '@3'. Alternatively, we could try using the same trick as with ecflow, i.e., use `bison::` in the site configs to override the common setting.